contribution/contributions.md

```diff
@@ -146,7 +146,12 @@ We are following Google Code style:
 * [Java style](https://google.github.io/styleguide/javaguide.html)
 * [Shell style](https://google.github.io/styleguide/shell.xml)
 
-Check style report location are in `${submodule}/target/site/checkstyle.html`
+There are some plugins to format, lint your code in IDE (use [_tools/checkstyle.xml](https://github.com/apache/zeppelin/tree/master/_tools) as rules)
+
+* [Checkstyle plugin for Intellij](http://eclipse-cs.sourceforge.net/#!/project-setup)
```

Comment:
@1ambda i mean `Checkstyle plugin for Intellij` has link for eclipse http://eclipse-cs.sourceforge.net/#!/project-setup, and the other way around